prebid-mobile/modules/rendering/ios-sdk-integration-admob.md

```diff
@@ -222,37 +210,37 @@ The process of displaying the rewarded ad is the same as for displaying an Inter
 
 To be notified when a user earns a reward follow the [AdMob intructions](https://developers.google.com/admob/ios/rewarded#show_the_ad).
 
-### Step 1: Create GADRequest
+#### Step 1: Create GADRequest
 {:.no_toc}
 
 This step is the same as for the original [AdMob integration](https://developers.google.com/admob/ios/rewarded). You don't have to make any modifications here.
 
-### Step 2: Create MediationRewardedAdUnit
+#### Step 2: Create MediationRewardedAdUnit
 {:.no_toc}
 
 The `AdMobMediationRewardedUtils` is a helper class, which performs certain utilty work for the `MediationRewardedAdUnit`, like passing the targeting keywords to the adapters.
 
-### Step 3: Create MediationInterstitialAdUnit
+#### Step 3: Create MediationInterstitialAdUnit
 {:.no_toc}
 
 The `MediationRewardedAdUnit` is part of the Prebid mediation API. This class is responsible for making a bid request and providing a winning bid and targeting keywords to the adapters.  
 
-### Step 4: Make bid request
+#### Step 4: Make bid request
 {:.no_toc}
 
 The `fetchDemand` method makes a bid request to the a Prebid server and provides a result in a completion handler.
 
-### Step 5: Make an Ad Request
+#### Step 5: Make an Ad Request
 {:.no_toc}
 
 Make a regular AdMob's ad request. Everything else will be handled by GMA SDK and prebid adapters.
 
-### Steps 6: Display an ad
+#### Steps 6: Display an ad
 {:.no_toc}
 
 Once the rewarded ad is received you can display it. Follow the [AdMob instructions](https://developers.google.com/admob/ios/rewarded#swift) for displaying an ad.
 
-## Native Ads
+### Native Ads
 
 {: .alert.alert-warning :}
 **Warning:** If you use Native Ads you **must** integrate AdMob Adapters via the source files instead of cocoapods integration or standalone framework. The integration using framework leads to [runtime errors](https://github.com/prebid/prebid-mobile-ios/issues/516) related to the type casting.
```

Comment:
Regarding the issue. It was closed with the respective comment that we had not found a fix but only a workaround - to add the adapter as a source code.

Reply:
Are you suggesting dropping the link to the issue? The warning already says to "integrate AdMob Adapters via the source files"

Reply:
I think we can keep it. In this way, someone can try to fix it too.